server/app/models/QuestionModel.java

```diff
@@ -109,6 +109,11 @@ public QuestionModel addVersion(VersionModel version) {
   @PreUpdate
   @PrePersist
   public void persistChangesToQuestionDefinition() {
+    // Play Ebeans starting at v6.2.0 includes updated Ebeans that fixes a bug we
+    // had relied on to mark the json fields as dirty. We now need to manually
+    // trigger the dirty flag or the @PrePersist/@PreUpdate annotations don't
+    // get triggered.
```

Comment:
We had to do the same thing with program data a while ago in order to get off of v6.2.0-beta-4. It's our own fault.